### ISSUE TITLE
returnUrl in LoginContent

### ISSUE THREAD

**Issue by felipe-ssilva** [vtex-apps/login#16]
Hi guys! How it's going?

How using returnUrl in LoginContent? This is possible?
The login is working. But returnUrl after login not.

```
import LoginContent from 'vtex.login/LoginContent'; 

 <LoginContent
          profile={profile}
          defaultOption={1}
          returnUrl={encodeURIComponent(urlLocal)}
          emailAndPasswordTitle={`Entrar com e-mail e senha:`}
          optionsTitle={`Use uma das opções abaixo:`}
          accessCodeTitle="Receber código de acesso por e-mail"
          emailPlaceholder="Digite seu e-mail"
          passwordPlaceholder="Digite sua senha"
          accessCodePlaceholder="Digite seu código"
          isInitialScreenOptionOnly={false}
          showPasswordVerificationIntoTooltip={true}
/>
```

Custom app:
![image](https://user-images.githubusercontent.com/4562886/172508940-c7154997-e377-45cd-be0b-179b938afe15.png)

**Comment by charzlwebz256** [comment]
Yes, it's possible to use the returnUrl prop in the LoginContent component to redirect the user to a specific page after they log in. Here's an example of how you can use it:
`import LoginContent from 'vtex.login/LoginContent';

const MyLoginPage = () => {
  const returnUrl = encodeURIComponent('https://example.com/my-account');

  return (
    <LoginContent
      profile={profile}
      defaultOption={1}
      returnUrl={returnUrl}
      emailAndPasswordTitle={`Entrar com e-mail e senha:`}
      optionsTitle={`Use uma das opções abaixo:`}
      accessCodeTitle="Receber código de acesso por e-mail"
      emailPlaceholder="Digite seu e-mail"
      passwordPlaceholder="Digite sua senha"
      accessCodePlaceholder="Digite seu código"
      isInitialScreenOptionOnly={false}
      showPasswordVerificationIntoTooltip={true}
    />
  );
};
`
In this example, the returnUrl prop is set to https://example.com/my-account, which is the URL that the user will be redirected to after they log in. Make sure to encode the URL using the encodeURIComponent function, as shown in the example.

When the user logs in, they will be redirected to the URL specified in the returnUrl prop. If the returnUrl prop is not specified or is set to an empty string, the user will be redirected to the home page of the website.

**Comment by Abongile184** [comment]
Yo
I don't know why am I receiving these mails please remove me from your
mailing list

**Comment by manutromero** [comment]
@charzlwebz256 But, how you imported vtex.login: "2.x" in you manifest? I got this error.

![image](https://github.com/vtex-apps/login/assets/7321696/90167da9-dee5-4795-9470-9e5c35700df3)

**Comment by nohestmm** [comment]
Hi, not work for me, did it work for anyone

**Comment by Eugod** [comment]
Didn't work for me either
